### PR TITLE
Add class ECLWellSolution

### PR DESCRIPTION
**Pull request by atgeirr**
The class reads well completion reservoir fluxes from a restart file, using ERT functions.

### CODE REVIEW

File: opm/utility/ECLWellSolution.cpp [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+        // Reservoir rate units from code used in INTEHEAD.
+        double resRateUnit(const int unit_code)
+        {
+            using prefix::centi;
+            using namespace unit;
+
+            switch (unit_code) {
+            case INTEHEAD::Metric: return cubic(meter)/day;    // m^3/day
+            case INTEHEAD::Field: return stb/day;              // stb/day
+            case INTEHEAD::Lab: return cubic(centi*meter)/day; // (cm)^3/day
```
Comment (atgeirr) [inline opm/utility/ECLWellSolution.cpp]: Good catch, thanks!

**Comment by bska** [comment]
I'm a little uneasy about exposing `ecl_grid.h` and, to a certain extent, `ert_unique_ptr.hpp` to all users of `ECLWellSolution.hpp`, but other than that this looks good.  If you're happy about its current state then I'll merge into master.

**Comment by atgeirr** [comment]
I am not done with this class, but adding the LGR features was a little more involved than first hoped, so I think it is best to merge now. It can be tested with the current feature set.

**Comment by bska** [comment]
> I think it is best to merge now

Very well.  Do you want to rebase on top of current master before I merge?

**Comment by atgeirr** [comment]
Rebased.

**Comment by bska** [comment]
> Rebased.

:+1: 

I'll merge into master.  Thanks a lot.

**Pull request merged**